Mathlib/AlgebraicGeometry/PrimeSpectrum/Basic.lean

```diff
@@ -927,8 +928,58 @@ def localizationMapOfSpecializes {x y : PrimeSpectrum R} (h : x ⤳ y) :
         ⟨a, show a ∈ x.asIdeal.primeCompl from h ha⟩ : _))
 #align prime_spectrum.localization_map_of_specializes PrimeSpectrum.localizationMapOfSpecializes
 
+variable (R) in
+/--
+Zero loci of prime ideals are closed irreducible sets in the Zariski topology and any closed
+irreducible set is a zero locus of some prime ideal.
+-/
+@[simps] protected def bijection :
+    PrimeSpectrum R ≃o {s : Set (PrimeSpectrum R) | IsClosed s ∧ IsIrreducible s}ᵒᵈ :=
+  { toFun := fun p ↦ ⟨zeroLocus p.asIdeal, isClosed_zeroLocus p.asIdeal,
+      isIrreducible_zeroLocus_iff _ |>.mpr <| by simpa only [p.IsPrime.radical] using p.IsPrime⟩
+    invFun := (fun s ↦ ⟨vanishingIdeal s.1, isIrreducible_iff_vanishingIdeal_isPrime.mp s.2.2⟩)
+    left_inv := fun p ↦ PrimeSpectrum.ext _ _ <|
+      by simp [PrimeSpectrum.vanishingIdeal_zeroLocus_eq_radical, p.IsPrime.radical]
+    right_inv := fun s ↦ Subtype.ext <|
+      by simpa [zeroLocus_vanishingIdeal_eq_closure] using s.2.1.closure_eq
+    map_rel_iff' := by
+      intro p q
+      change zeroLocus _ ≤ zeroLocus _ ↔ _
+      simp [zeroLocus_subset_zeroLocus_iff, q.IsPrime.radical] }
+
 end PrimeSpectrum
 
+open PrimeSpectrum in
+/--
+[Stacks: Lemma 00ES (3)](https://stacks.math.columbia.edu/tag/00ES)
+Zero loci of minimal prime ideals of `R` are irreducible components in `Spec R` and any
+irreducible component is a zero locus of some minimal prime ideal.
+-/
+protected def minimalPrimes.bijection :
```

Comment:
I think something like `zero_locus ∘ coe '' minimal_primes R = irreducible_components (prime_spectrum R)` or `vanishing_ideal '' irreducible_components (prime_spectrum R) = minimal_primes R` could be useful.

Reply:
Moogle found [irreducibleComponents_eq_maximals_closed](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Irreducible.html#irreducibleComponents_eq_maximals_closed). So this should follow from a general construction that an antitone OrderIso sends `minimals` to `maximals`.

Reply:
the equality has been added